target/linux/lantiq/image/Makefile

```diff
@@ -668,84 +720,97 @@ TARGET_DEVICES += avm_fritz7360sl
 define Device/avm_fritz7412
   $(Device/AVM)
   $(Device/NAND)
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 7412
   BOARD_NAME := FRITZ7412
   DEVICE_DTS := FRITZ7412
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 49152k
-  DEVICE_TITLE := AVM FRITZ!Box 7412
   DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-mini kmod-ltq-tapi kmod-ltq-vmmc fritz-tffs-nand fritz-caldata
 endef
 TARGET_DEVICES += avm_fritz7412
 
 define Device/avm_fritz7362sl
   $(Device/AVM)
   $(Device/NAND)
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 7362 SL
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 49152k
   DEVICE_DTS := FRITZ7362SL
-  DEVICE_TITLE := AVM FRITZ!Box 7362 SL
   DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-basic kmod-usb-dwc2 fritz-tffs
 endef
 TARGET_DEVICES += avm_fritz7362sl
 
 define Device/arcadyan_vg3503j
+  DEVICE_VENDOR := BT Openreach
+  DEVICE_MODEL := ECI
+  DEVICE_VARIANT := VDSL Modem V-2FUb/R
   IMAGE_SIZE := 8000k
   DEVICE_DTS := VG3503J
-  DEVICE_TITLE := BT OpenReach - ECI VDSL Modem V-2FUb/R
   SUPPORTED_DEVICES += VG3503J
 endef
 TARGET_DEVICES += arcadyan_vg3503j
 
 include tp-link.mk
 
 define Device/buffalo_wbmr-300hpd
+  DEVICE_VENDOR := Buffalo
+  DEVICE_MODEL := WBMR-300HPD
   IMAGE_SIZE := 15616k
   DEVICE_DTS := WBMR300
-  DEVICE_TITLE := Buffalo WBMR-300HPD
   DEVICE_PACKAGES := kmod-mt7603 wpad-basic kmod-usb-dwc2
   SUPPORTED_DEVICES += WBMR300
 endef
 TARGET_DEVICES += buffalo_wbmr-300hpd
 
 define Device/arcadyan_vgv7510kw22-nor
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := Box 6431 / Arcadyan
+  DEVICE_VARIANT := VGV7510KW22 (NOR)
   IMAGE_SIZE := 15232k
   DEVICE_DTS := VGV7510KW22NOR
-  DEVICE_TITLE := o2 Box 6431 / Arcadyan VGV7510KW22 (NOR)
   DEVICE_PACKAGES := kmod-rt2800-pci wpad-basic kmod-usb-dwc2 kmod-ltq-tapi kmod-ltq-vmmc
   SUPPORTED_DEVICES += VGV7510KW22NOR
 endef
 TARGET_DEVICES += arcadyan_vgv7510kw22-nor
 
 define Device/arcadyan_vgv7510kw22-brn
   $(Device/lantiqBrnImage)
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := Box 6431 / Arcadyan
+  DEVICE_VARIANT := VGV7510KW22 (BRN)
   IMAGE_SIZE := 7168k
   DEVICE_DTS := VGV7510KW22BRN
   SIGNATURE := BRNDA6431
   MAGIC := 0x12345678
   CRC32_POLY := 0x04c11db7
-  DEVICE_TITLE := o2 Box 6431 / Arcadyan VGV7510KW22 (BRN)
   DEVICE_PACKAGES := kmod-rt2800-pci wpad-basic kmod-usb-dwc2 kmod-ltq-tapi kmod-ltq-vmmc
   SUPPORTED_DEVICES += VGV7510KW22BRN
 endef
 TARGET_DEVICES += arcadyan_vgv7510kw22-brn
 
 define Device/arcadyan_vgv7519-nor
+  DEVICE_VENDOR := KPN
+  DEVICE_MODEL := Experiabox 8
+  DEVICE_VARIANT := VGV7519
   IMAGE_SIZE := 15360k
   DEVICE_DTS := VGV7519NOR
-  DEVICE_TITLE := Experiabox 8 VGV7519
   DEVICE_PACKAGES := kmod-rt2800-pci wpad-basic kmod-usb-dwc2 kmod-ltq-tapi kmod-ltq-vmmc
   SUPPORTED_DEVICES += VGV7519NOR
 endef
 TARGET_DEVICES += arcadyan_vgv7519-nor
 
 define Device/arcadyan_vgv7519-brn
   $(Device/lantiqBrnImage)
+  DEVICE_VENDOR := KPN
+  DEVICE_MODEL := Experiabox 8
+  DEVICE_VARIANT := VGV7519 (BRN)
```

Comment:
Same here. VARIANT should be `BRN`, to be consistent with other Arcadyan boards above, maybe use
```
DEVICE_VENDOR := Arcadyan
DEVICE_MODEL := KPN Experiabox 8 - VGV7519
DEVICE_VARIANT := BRN
```

Reply:
Having learned that this seems to be marketed name and ODM name, I would suggest to always use the marketed name and then put the ODM name in brackets, e.g. `KPN Experiabox 8 (VGV7519)`.
By this, both would be visible to the user, but it would still be fit into some kind of system.

In the long run, one could think about providing an additional field for the ODM name.

Reply:
Resolved with *_ALT* variables.